main.tf

```diff
@@ -193,7 +193,7 @@ data "aws_iam_policy_document" "cloudtrail_kms_policy_doc" {
     condition {
       test     = "StringEquals"
       variable = "kms:CallerAccount"
-      values   = [data.aws_caller_identity.current.account_id]
+      values   = compact(concat([data.aws_caller_identity.current.account_id], var.key_allowed_extra_accounts))
```

Comment:
How are you testing this? Reading [AWS docs](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/default-cmk-policy.html), don't you also need to include the additional account is in the conditional on line [202](https://github.com/trussworks/terraform-aws-cloudtrail/pull/182/files#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbR202)